docker/InternVL_Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG CUDA_VERSION=cu12
+
+FROM openmmlab/lmdeploy:latest-cu12 AS cu12
+
+FROM openmmlab/lmdeploy:latest-cu11 AS cu11
+
```

Comment:
this dockerfile is too short and simple, should we keep it in the repo?

Reply:
> this dockerfile is too short and simple, should we keep it in the repo? could put it in document

lvhan028 hope every VLM model have their own Dockerfile at the  future,some LLM Don't need timm and flash-attn,but vlm like internVL2 and minicpm need it;the InternVL_Dockerfile is internvl's Dockerfile,if InternVL have more dependencies , we can add it in  InternVL_Dockerfile without  contaminate base Dockerfile